orbit-2-core/src/main/java/com/babylon/orbit2/RealContainer.kt

```diff
@@ -104,12 +108,14 @@ open class RealContainer<STATE : Any, SIDE_EFFECT : Any>(
             }.collect()
     }
 
-    companion object {
+    private companion object {
```

Comment:
By making this `private` we lose the ability to be able to add extension functions to RealContainer outside of this file.